backend/config/locales/zu.yml

```diff
@@ -40,6 +40,15 @@ zu:
         multiple_accounts: User already have account!
         no_project_developer: User is not project developer!
         no_investor: User is not investor!
+  user_mailer:
+    greetings: Dear %{full_name},
+    farewell_html: Best Regards,<br />Your HeCo Team
+    approved:
+      subject: Your account was approved!
+      content: We are happy to inform you that your account at HeCo platform was approved.
+    rejected:
+      subject: Your account was rejected!
+      content_html: We would like to inform you that your account at HeCo platform was rejected. One of our platform administrators should reach you shortly, but feel free to contact us at %{contact_url}.
```

Comment:
I would be probably more lazy in terms of making up the content of this email :P

Reply:
:) yeah, not sure if I was not overthinking this, but I am open to put here whatever text you feel would be more appropriate